ngraph/core/reference/src/runtime/reference/if.cpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "ngraph/runtime/reference/if.hpp"
+#include "ngraph/runtime/reference/function.hpp"
+namespace ngraph
+{
+    namespace runtime
+    {
+        namespace reference
+        {
+            enum if_body_indexes
```

Comment:
Can we reuse enum from the operation?

Reply:
Yes, we can use it, I thought include if operation in reference is bad idea